color/src/colorspace.rs

```diff
@@ -165,6 +167,12 @@ impl ColorSpace for LinearSrgb {
     }
 }
 
+/// The standard RGB color space (IEC 61966-2-1).
+///
+/// Its components are `[r, g, b]` (red, green, and blue channels respectively), with `[0, 0, 0]`
+/// pure black, and `[1, 1, 1]` reference white (CIE D65).
+///
+/// The natural bounds of the channels are `[0, 1]`.
 // It might be a better idea to write custom debug impls for AlphaColor and friends
```

Comment:
Please also remove this comment; we may well want to rethink the Debug impls, but using the derive is fine for now.